nion/swift/LinePlotCanvasItem.py

```diff
@@ -430,35 +432,66 @@ def __view_to_intervals(self, data_and_metadata: DataAndMetadata.DataAndMetadata
         else:
             left = right = 0.0
```

Comment:
This line leads to undesired behavior - it should set `left = 0.0` and `right = 1.0`. Can you add a test for this? Hitting <enter> with no interval selected should reset the view to its default.